src/applications/vaos/utils/appointment.jsx

```diff
@@ -371,12 +372,12 @@ export function getAppointmentInstructions(appt) {
       return appt.instructionsToVeteran;
     case APPOINTMENT_TYPES.vaAppointment: {
       const bookingNotes =
-        appt.vdsAppointments?.[0]?.bookingNote ||
-        appt.vvsAppointments?.[0]?.bookingNotes;
+        appt.vdsAppointments[0]?.bookingNote ||
```

Comment:
Why did you remove the `?.` after `vdsAppointments`? Those keeps us from throwing errors if an object without `vdsAppointments` is passed to this function.

Reply:
Added the optional chaining operator back. However, I did find that this feature is not supported by all browsers.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
All of our JS is run through Babel, which we use to transform JS with more modern features into JS that can be used by all the browsers we support. We have it set up to transform optional chaining (https://github.com/department-of-veterans-affairs/vets-website/blob/master/.babelrc#L32), so we should be good.

This is a good thing to think about whenever you're using new features, though. Things that are features of the "web platform" (like the `download` attribute I commented on below, or new browser apis) aren't always something we can handle through Babel.